src/modules/application/components/HomePage/HomePage.jsx

```diff
@@ -18,7 +20,9 @@ const HomePage = () => {
 
     return (
         <>
-            <FilterBar />
+            <FilterFetcher>
```

Comment:
For the best it should be just <Filters /> here and inside <Filters /> component should be this, to save space in Homepage component

Reply:
Done in #11